Makefile

```diff
@@ -0,0 +1,9 @@
+CC=gcc
+CFLAGS=-lmraa
```

Comment:
`CFLAGS` is the commonly used variable for compile time flags. But `lmraa` is the link time flag. So better change it to `LFLAGS`.

Reply:
LFLAGS causes compilation error.
all mraa functions resulting in "undefined reference"